charts/openproject/templates/_helpers.tpl

```diff
@@ -37,11 +37,46 @@ securityContext:
 {{- end }}
 
 {{- define "openproject.useTmpVolumes" -}}
-{{- if (default .Values.useTmpVolumes (not .Values.develop)) -}}
+{{- $useTmpVolumes := .Values.openproject.useTmpVolumes | toString | lower }}
+{{- if and (not (eq $useTmpVolumes "false")) (not .Values.develop) -}}
```

Comment:
I would prefer it if would still allow using tmp volums even when in dev mode here like before.